hack/arktos-up-scale-out-poc.sh

```diff
@@ -659,7 +659,7 @@ echo "Setup Arktos components ..."
 echo ""
 
 # todo: start flannel daemon deterministically, instead of waiting for arbitrary time
-if [[ "${CNIPLUGIN}" == "flannel" && "${IS_RESOURCE_PARTITION}" == "true" ]]; then
+if [[ "${CNIPLUGIN}" == "flannel" && "${IS_RESOURCE_PARTITION}" == "true" && "${SCALE_OUT_TP_ENABLE_DAEMONSET}" == "false" ]]; then
```

Comment:
can we leverage ${ARKTOS_NO_CNI_PREINSTALLED} to sort this logic out? the intention of this var is NOT to install any CNI plugin during the installation process.

Reply:
I need discuss this logic with you before I move forward.

Reply:
Hongwei suggests to temporarily ignore DaemonSet feature to install Flannel. so I will remove check the setting of variable ${SCALE_OUT_TP_ENABLE_DAEMONSET}.